web/src/test/java/no/nav/foreldrepenger/fordel/web/app/konfig/RestApiAbacTest.java

```diff
@@ -99,9 +99,9 @@ public void test_at_alle_input_parametre_til_restmetoder_implementer_AbacDto() t
         if (annotation != null && annotation.action() == BeskyttetRessursActionAttributt.DUMMY) {
             fail(klasse.getSimpleName() + "." + metode.getName() + " Ikke bruk DUMMY-verdi for "
                     + BeskyttetRessursActionAttributt.class.getSimpleName());
-        } else if (annotation != null && annotation.ressurs() == BeskyttetRessursResourceAttributt.DUMMY) {
+        } else if (annotation != null && annotation.resource() == BeskyttetRessursAttributt.DUMMY) {
```

Comment:
Her kan man nok droppe DUMMY sjekk og sjekke for tom string isteden.
https://github.com/navikt/fp-abakus/blob/1b038bd67483241e9b62fb442070a2aa1d54acfc/web/src/test/java/no/nav/foreldrepenger/abakus/app/konfig/RestApiAbacTest.java#L106